### ISSUE TITLE
Random duplicates volume icons

### ISSUE THREAD

**Issue by unixfox** [johnhoran/Laine#36]
This bug appears the most of the time and randomly when I lock my session then my computer goes on standby. When I unlock my session Gnome freeze for around 10 minutes and lot's of volume icons appears:
![](https://i.unixfox.eu/44-12-44-2016.png)
I think it's related to the last update because I never had this problem before.

**Comment by johnhoran** [comment]
Sorry about that, definitely related to the update.  I'll look into it and figure out whats happening.  Can you tell me what version of gnome you are running and see if anything was visible in the logs?

**Comment by unixfox** [comment]
Thank you for the reply.
I'm using Gnome 3.20.2 on Archlinux:
![](https://i.unixfox.eu/56-15-49-2016.png)
Where can I get logs of Gnome 3 ?

**Comment by johnhoran** [comment]
Hmm, I'm running exactly the same and I haven't run into this.

You can see the logs in journalctl by running `journalctl -e /usr/bin/gnome-shell` that will also filter them a bit.  Also, you said that gnome shell was freezing for 10 minutes, so if its possible to ssh in from another computer and see if there are any repeating lines showing up in your log that might point at that issue while gnome is frozen.

This is a pretty terrible bug so I'm going to inactivate this version on the gnome-extensions site, I've never done that before so I don't know what happens, but I hope maybe people can downgrade.  But please see if you can get the logs for me before you do so.

**Comment by unixfox** [comment]
Okay I'll try to catch logs when I'll have the bug.

**Comment by rickysarraf** [comment]
I have been hit by the same issue. But my symptoms are slightly different.

* They only happen on system startup.
* If I do a shell restart, it goes away.

When I did a restart, I watched the logs. But I don't see any serious errors from Laine.

```

Feb 11 19:03:39 learner gnome-shell[10365]: gdk_window_set_user_time called on non-toplevel
Feb 11 19:03:39 learner gnome-shell[10365]: gdk_window_set_user_time called on non-toplevel
Feb 11 19:03:39 learner org.gnome.Shell.desktop[10365]: Window manager warning: CurrentTime used to choose focus window; focus window may not be correct.
Feb 11 19:03:41 learner org.gnome.Shell.desktop[10365]: current session already has an ibus-daemon.
Feb 11 19:03:41 learner gnome-shell[10365]: JS LOG: Failed to add search provider /usr/share/gnome-shell/search-providers/org.gnome.bijiben-search-provider.ini: TypeError: appInfo is null
Feb 11 19:03:43 learner gnome-shell[10365]: JS LOG: System monitor applet init from /home/rrs/.local/share/gnome-shell/extensions/system-monitor@paradoxxx.zero.gmail.com
Feb 11 19:03:43 learner gnome-shell[10365]: JS LOG: System monitor applet enabling
Feb 11 19:03:43 learner gnome-shell[10365]: JS LOG: System monitor applet enabling done
Feb 11 19:03:44 learner gnome-shell[10365]: JS WARNING: [/home/rrs/.local/share/gnome-shell/extensions/laine@knasher.gmail.com/extension.js 374]: applying the 'delete' operator to an unqualified name is deprecated
Feb 11 19:03:44 learner gnome-shell[10365]: JS LOG: Extension alternate-tab@gnome-shell-extensions.gcampax.github.com already installed in /home/rrs/.local/share/gnome-shell/extensions/alternate-tab@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/alternate-tab@gnome-shell-extensions.gcampax.github.com will not be loaded
Feb 11 19:03:44 learner gnome-shell[10365]: JS LOG: Extension apps-menu@gnome-shell-extensions.gcampax.github.com already installed in /home/rrs/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com will not be loaded
Feb 11 19:03:44 learner gnome-shell[10365]: JS LOG: Extension auto-move-windows@gnome-shell-extensions.gcampax.github.com already installed in /home/rrs/.local/share/gnome-shell/extensions/auto-move-windows@gnome-shell-extensions.gcampax.github.com. /usr/share/gnome-shell/extensions/auto-move-windows@gnome-shell-extensions.gcampax.github.com will not be loaded
Feb 11 19:03:44 learner gnome-shell[10365]: JS LOG: loading default theme (Adwaita)
Feb 11 19:03:46 learner org.gnome.Shell.desktop[10365]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x0 specified for 0x1c00007 (Onboard).
Feb 11 19:03:46 learner gnome-shell[10365]: JS LOG: [Dynamic Panel Transparency] No background image found in user theme.
Feb 11 19:03:46 learner gnome-shell[10365]: JS LOG: [Dynamic Panel Transparency] No border image found in user theme.
Feb 11 19:03:46 learner gnome-shell[10365]: JS LOG: [Dynamic Panel Transparency] Detected user theme style: rgba(0, 0, 0, 255)
Feb 11 19:03:47 learner gnome-shell[10365]: GNOME Shell started at Sat Feb 11 2017 19:03:41 GMT+0530 (IST)
                        
```

**Comment by johnhoran** [comment]
Should be fixed in 951579397185fa8c6e256571549e074d9a0fae28
Closing unless somebody notices it happens still.
